ledger/src/shred/shred_code.rs

```diff
@@ -14,8 +14,7 @@ use {
 };
 
 const_assert_eq!(MAX_CODE_SHREDS_PER_SLOT, 32_768 * 17);
-pub(crate) const MAX_CODE_SHREDS_PER_SLOT: usize =
-    MAX_DATA_SHREDS_PER_SLOT * (ERASURE_BATCH_SIZE[1] - 1);
+pub const MAX_CODE_SHREDS_PER_SLOT: usize = MAX_DATA_SHREDS_PER_SLOT * (ERASURE_BATCH_SIZE[1] - 1);
```

Comment:
Why is this modified?
Also I am reducing this in https://github.com/solana-labs/solana/pull/30543

Reply:
I just made it pub so I can import it into the other crate to check for excess coding shreds.

Reply:
Value is not changed.

Reply:
sounds like the test (of the constant values) is in the wrong place, then?

Reply:
sorry, are you saying move the MAX_CODE_SHREDS_PER_SLOT into the core crate?

Reply:
nah. that the test in core doesn't really care what the values are and that if a more specific test around the constants is warranted, it should be in the ledger crate

Reply:
Yes, the test code doesn't use the constant at all, just in the validator code when I call that function I use the real constant value.

Reply:
ah... derp. sorry misread as in test there
![Screenshot from 2023-03-03 16-51-20](https://user-images.githubusercontent.com/490004/222856676-23734c9f-ecaa-4059-96df-f03502199973.png)